catalog/imaging/imaging-plugin-nitf/src/test/java/org/codice/alliance/plugin/nitf/PreStoragePluginMultiTest.java

```diff
@@ -1,163 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- *
- * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
- * Lesser General Public License as published by the Free Software Foundation, either version 3 of
- * the License, or any later version.
- *
- * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
- * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
- * License is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- */
-package org.codice.alliance.plugin.nitf;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.notNullValue;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-import ddf.catalog.content.data.ContentItem;
-import ddf.catalog.content.operation.CreateStorageRequest;
-import ddf.catalog.content.operation.UpdateStorageRequest;
-import ddf.catalog.data.Attribute;
-import ddf.catalog.data.Metacard;
-import ddf.catalog.data.types.Core;
-import ddf.catalog.plugin.PluginExecutionException;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.List;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.ArgumentCaptor;
-
-/** Test the PreStoragePlugin with multiple content items. */
```

Comment:
Now that this class is deleted, are there any tests for `CreateResponse`s\`UpdateResponse`s with multiple `Metacards`?

Reply:
I could add one in the other class if you think it adds value.

Reply:
I think it does add value. I think what's important here is that other metacards and images still get processed even if there is an exception during processing of one of the metacards.

Reply:
Added a test with two metacards where the first one will fail processing.